src/components/View.js

```diff
@@ -2,47 +2,17 @@
  * https://github.com/facebook/react-native/blob/master/Libraries/Components/View/View.js
  */
 import React from 'react';
-import styleSheetPropType from '../propTypes/StyleSheetPropType';
-import ViewStylePropTypes from '../propTypes/ViewStylePropTypes';
+import ViewAccessibility from './ViewAccessibility';
 import NativeMethodsMixin from '../mixins/NativeMethodsMixin';
+import ViewPropTypes from '../propTypes/ViewPropTypes';
 
-const stylePropType = styleSheetPropType(ViewStylePropTypes);
+const { AccessibilityTraits, AccessibilityComponentTypes } = ViewAccessibility;
 
-const { PropTypes } = React;
-
-const AccessibilityTraits = [
-  'none',
-  'button',
-  'link',
-  'header',
-  'search',
-  'image',
-  'selected',
-  'plays',
-  'key',
-  'text',
-  'summary',
-  'disabled',
-  'frequentUpdates',
-  'startsMedia',
-  'adjustable',
-  'allowsDirectInteraction',
-  'pageTurn',
-];
-
-const AccessibilityComponentType = [
-  'none',
-  'button',
-  'radiobutton_checked',
-  'radiobutton_unchecked',
-];
-
-// TODO(lmr):
 const forceTouchAvailable = false;
 
 const statics = {
+  AccessibilityComponentType: AccessibilityComponentTypes,
```

Comment:
There's a plurality difference here unfortunately. I made the decision to follow the pattern in ```react-native```. However, I'm not super insightful about this property in ```statics```, so I'm open to ideas.